packages/config/config/devices/0x0258/nas-pd07z.json

```diff
@@ -10,63 +10,169 @@
 		}
 	],
 	"firmwareVersion": {
-		"min": "0.0",
+		"min": "2.6",
 		"max": "255.255"
 	},
 	"paramInformation": [
 		{
 			"#": "1",
-			"$import": "templates/shenzhen_neo_template.json#blink_on_motion"
+			"label": "Led indicator enable",
+			"valueSize": 1,
+			"defaultValue": 1,
+			"unsigned": true,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
 		},
 		{
 			"#": "2",
-			"$import": "~/templates/master_template.json#base_enable_disable",
 			"label": "Motion Detection",
-			"defaultValue": 1
+			"valueSize": 1,
+			"unsigned": true,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
```

Comment:
For example, this change does nothing except replace the template import with its "full" representation